tests/sentry/api/serializers/test_event.py

```diff
@@ -100,6 +100,9 @@ def test_message_legacy(self):
 
         # create_event automatically creates the logentry interface
         del event.data['logentry']
+        # need to re-save because the serializer forces a re-fetch from
+        # nodestore which will reanmiate the property if we don't save.
```

Comment:
Is 'reanmiate' supposed to be 'reanimate'?

Reply:
yep